UI/MiscScreens.cpp

```diff
@@ -798,7 +799,9 @@ void LogoScreen::render() {
 	// Draw the graphics API, except on UWP where it's always D3D11
 	std::string apiName = screenManager()->getDrawContext()->GetInfoString(InfoField::APINAME);
 #ifdef _DEBUG
-	apiName += ", debug build";
+	apiName += ", debug build ";
+	// Add some emoji for testing.
+	apiName += CodepointToUTF8(0x1F41B) + CodepointToUTF8(0x1F41C) + CodepointToUTF8(0x1F914);
```

Comment:
Did you mean for this to stay?  I mean, it's fine, just noting in case.

-[Unknown]

Reply:
yes, since it's debug build only :)